app/queries/payments_requiring_action.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class PaymentsRequiringAction
+  def initialize(user)
+    @user = user
+  end
+
+  attr_reader :user
+
+  def query
+    Spree::Payment.joins(order: [:user]).where.not(cvv_response_message: nil).
+      where("spree_users.id = ?", user.id)
+  end
+end
```

Comment:
This should be unit tested IMO.

Reply:
bccdcc373cdd2abb0e3210a7749ec1e1e1574da0